bundler/spec/commands/newgem_spec.rb

```diff
@@ -698,6 +698,33 @@ def create_temporary_dir(dir)
         to match(/example\.com/)
     end
 
+    context "git config github.user present" do
+      before do
+        bundle "gem #{gem_name}"
+      end
+
+      it "gemspec changelog_uri set to include git username" do
+        expect(generated_gemspec.metadata["changelog_uri"]).
+          to eq("https://github.com/bundleuser/#{gem_name}/blob/main/CHANGELOG.md")
+        expect(generated_gemspec.metadata["homepage_uri"]).to eq("https://github.com/bundleuser/#{gem_name}")
+        expect(generated_gemspec.metadata["source_code_uri"]).to eq("https://github.com/bundleuser/#{gem_name}")
+      end
+    end
+
+    context "git config github.user is absent" do
+      before do
+        git("config --global --unset github.user")
+        bundle "gem #{gem_name}"
+      end
+
+      it "sets gemspec changelog_uri to TODO" do
+        expect(generated_gemspec.metadata["changelog_uri"]).
+        to eq("TODO: Put your gem's CHANGELOG.md URL here.")
```

Comment:
minor indentation again.
```suggestion
          to eq("TODO: Put your gem's CHANGELOG.md URL here.")
```